dbms/src/Flash/Mpp/MPPTask.cpp

```diff
@@ -412,7 +412,11 @@ void MPPTask::runImpl()
         while (from->read())
             continue;
 
+        // finish DataStream
         from->readSuffix();
+        // finish receiver
+        receiver_set->close();
```

Comment:
what if a error throwed before

Reply:
Then the error handle flow will abort the tunnels.